sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/MetricDataMapper.java

```diff
@@ -151,7 +151,11 @@ public static void updateMetricPointBuilder(
         pointBuilder.setValue(pointDataValue);
         // TODO (heya) why give it the same name as otel metric?
         //  it seems this field doesn't matter and only _MS.MetricId property matters?
-        pointBuilder.setName(metricData.getName());
+        if (metricData.getInstrumentationScopeInfo().getName().equals("jmx")) {
```

Comment:
can you add a comment for "To emit JMX metrics with spaces in the name to Breeze, we need to use the schema url as the metric name that is reported to Breeze."?